packages/visual-tests/src/checkbox-group.visual.spec.js

```diff
@@ -15,7 +15,7 @@ describe('CheckboxGroup', () => {
     });
     test.each([['standard']])('%p', async (variant) => {
       await global.runSetup(`components-checkbox-group--${variant}`);
-      await global.page.waitForSelector('html.hydrated');
+      await global.page.waitForSelector('#root');
```

Comment:
Based on the pipeline results, There is a timing issue with the tests loading of the screen and I think in some places, therefore the `#root` selector is not enough. 
IMO we should wait for the complete element; 
`      await global.page.waitForSelector('#root > scale-checkbox-group > scale-checkbox:nth-child(1) > input[type=checkbox]');
`